src/main/preference.js

```diff
@@ -1,9 +1,21 @@
 import fs from 'fs'
 import path from 'path'
-import { ipcMain, BrowserWindow } from 'electron'
-import { isOsx } from './config'
+import { ipcMain, BrowserWindow, systemPreferences } from 'electron'
+import { isOsx, isWindows } from './config'
 import appWindow from './window'
 import { getPath, hasSameKeys, log, ensureDir } from './utils'
+import { getStringRegKey, winHKEY } from './platform/win32/registry.js'
+
+const isDarkSystemMode = () => {
+  if (isOsx) {
+    return systemPreferences.isDarkMode()
+  } else if (isWindows) {
+    // NOTE: This key is a 32-Bit DWORD but converted to JS string!
+    const buf = getStringRegKey(winHKEY.HKCU, 'Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize', 'AppsUseLightTheme')
```

Comment:
Where to set the dark mode of the Windows system on a Windows computer?

Reply:
![image](https://user-images.githubusercontent.com/9712830/55845136-37614900-5b73-11e9-98e3-c4910128e756.png)

is this?

Reply:
You can enable a dark theme since v1809: `Ctrl+I -> Personalization -> Colors` and then scroll down and select `Dark` under `Default application mode` (freely translated).

Reply:
haha, I found it.:smile: